test/trento_web/plugs/app_jwt_auth_plug_test.exs

```diff
@@ -132,9 +133,9 @@ defmodule TrentoWeb.Plugs.AppJWTAuthPlugTest do
 
   describe "create/3" do
```

Comment:
When you add the abilities also to the renew token add some test cases to assert that

Reply:
Right, at the end I just added an assertion to the same test 259cd3a16e91597602c7abe0961fff0b1ce47d61